controllers/tasks.js

```diff
@@ -48,8 +48,10 @@ const editTask = async (req, res) => {
 const updateTask = async (req, res) => {
   let task = false;
   try {
-    if (req.body.complete) {
+    if (req.body.isTaskComplete === "true") {
       req.body.completed = true;
+    } else {
+      req.body.completed = false;
     }
```

Comment:
If we want, this can be further simplified to: `req.body.completed = req.body.isTaskComplete === "true";`